src/main/java/com/groupseven/pdfproject/PageModel.java

```diff
@@ -38,11 +38,18 @@ public PageModel(BufferedImage image) {
         bufferedImage = image;
         Image fximage = SwingFXUtils.toFXImage(image, null);
         
-        canvas = new MainCanvas(fximage.getWidth(), fximage.getHeight(), fximage);
+        canvas = new MainCanvas(fximage.getWidth(), fximage.getHeight());
         
-        graphicsContext = canvas.getGraphicsContext();
+        graphicsContext = canvas.getCanvas().getGraphicsContext2D();
         
         node = new VBox(0);
+        BackgroundImage backgroundImage = new BackgroundImage(
+                fximage,
+                BackgroundRepeat.NO_REPEAT,
+                BackgroundRepeat.NO_REPEAT,
+                BackgroundPosition.CENTER,
```

Comment:
Using bp center causes issues when resizing the window, the drawing on the canvas is aligned to the left and top, while the image is now centered.

To make them consistent you can create a new left aligned background posititon:

```java
BackgroundPosition bp = new BackgroundPosition(Side.LEFT, 0, false, Side.TOP, 0, false);
BackgroundImage backgroundImage = new BackgroundImage(
                fximage,
                BackgroundRepeat.NO_REPEAT,
                BackgroundRepeat.NO_REPEAT,
                bp,
                BackgroundSize.DEFAULT);
```